packages/@dcl/inspector/src/lib/babylon/decentraland/sdkComponents/gltf-container.ts

```diff
@@ -8,7 +8,7 @@ import type { ComponentOperation } from '../component-operations'
 import { EcsEntity } from '../EcsEntity'
 import { SceneContext } from '../SceneContext'
 
-const sceneContextMap = new Map<string /*sceneId*/, WeakRef<SceneContext>>()
+let sceneContext: WeakRef<SceneContext>
```

Comment:
This was converted into a single reference instead of a map, since the scene id is always the same for the inspector, this came from a copypaste from the babylon explorer